### DISCUSSION TITLE
New user and new Two Trees CH340 5.5 W equipment. Only "chatter" response

### DISCUSSION THREAD

**Discussion by LaserFrallan** [arkypita/LaserGRBL#1657] [Q&A]
Hello,
Maybe I am in the wrong place but I will start here after looking at videos, google search on my issue and looking for support forums.
I have assembled, followed the manual exactly, installed software from download, installed CH340 driver, configured .nc file through Grbl Configuration.
The PC is Windows 10, the port is immediately recognized, even if I switch between the three available.
I connect the unit and I get a "flash" conformation and the drop down confirms that I am connected.
I unlock the unit and my bottom menu buttons all light up.
My first attempt is to get blink and focus. The laser will not light up.
My second attempt is to move the axis with the home and manual buttons. One or more very short motor reactions can be heard and then I get G code alarms as per below pictures.
i.e. "Abort during cycle".
The software locks up and only by resetting, can I repeat and try again.
The wiring is not possible to switch as it is both color and number coded plus ofcourse, the plugs are different depending on which motor is attached.
I have also re-installed once. I have restarted the PC a couple of times.
I have checked that the PC settings page can see and recognize that the device is connected, as USB Serial device CH340 and the COM port currently used.

Any suggestions?  Thanks!

![1](https://user-images.githubusercontent.com/94539219/142205933-6ed9f43d-6d35-4e35-a97f-b8bcf37d6a47.png)
![2](https://user-images.githubusercontent.com/94539219/142205938-df9fe4c0-bcb5-4197-b7c3-c6fe07216e58.png)
![3](https://user-images.githubusercontent.com/94539219/142205941-7a226f28-3e01-46a9-a63a-3524fa35e1c2.png)

**Comment by gmmanonymus111** [comment]
Could you post your `$$` GRBL setings? Steppers locking up can be caused either by bad configuration, or some hardware issues, like wrong wiring or noisy connection. (Since I did not find the exact model of your engraver, I've assumed it's a `Twotrees TT-5.5 Laser Engraver 400x300` one, which is, by user reviews, capable of having messed up wiring)
`Abort during cycle` could mean:
- You ran the laser head into a limit switch
- The hardware thinks you hit a limit switch because of electrical interference
- The hardware browned out or lost power briefly

It's worth to check your USB cable, along with the PSU, and if possible, try again with different USB cable and PSU. (PSU should be at least 3A since the laser is a 20W one (5.5W optical power).

For laser focusing, could you try the following commands? (Right click on the `Focus` button, then select `Edit button`)
![image](https://user-images.githubusercontent.com/9658243/142396728-ae34e6c1-cb83-4516-8d09-aa4786b2c19a.png)
If it does not light up, try increasing the `S` value from `S1` to e.g. `S10`.

**Comment by LaserFrallan** [comment]
Wow Alex!

 

You are a real source of knowledge!

I will look in to this soonest and respond.

As for the power supply, it is 60W 5A.

 

A colleague of mine who just yesterday ordered a unit himself, supports me and did same questions on a FB forum.

I will make 100% sure our findings are shared, on both sites, once we have something.

 

Kind regards, Fredrik

**Comment by LaserFrallan** [comment]
I begin by posting the Grbl settings.
![4](https://user-images.githubusercontent.com/94539219/142418355-ef96fa78-024d-44c4-b5de-13f153b1e33a.png)
![5](https://user-images.githubusercontent.com/94539219/142418358-46718f14-4b2a-4d27-bc2e-e8f34fc6907a.png)
![6](https://user-images.githubusercontent.com/94539219/142418359-d33d398a-ff1b-4a21-a01a-a902c74c3e30.png)
![7](https://user-images.githubusercontent.com/94539219/142418361-5d552103-18cf-4a84-bbd0-98d8883b4e84.png)
![8](https://user-images.githubusercontent.com/94539219/142418363-bad158c2-4178-489f-a65c-2aa85eb7ed64.png)
![9](https://user-images.githubusercontent.com/94539219/142418364-0f0a4694-d3ac-472c-92c1-619a6ba6cf04.png)

**Reply by gmmanonymus111** [reply]
I meant GRBL settings, not LaserGRBL (`Grbl`->`Grbl Configuration`) settings, like:
![image](https://user-images.githubusercontent.com/9658243/142419392-bce7a495-deec-43eb-b07a-fef341bc0bba.png)
or by typing `$$` into LaserGRBL's command input, then pressing enter:
![image](https://user-images.githubusercontent.com/9658243/142419731-88bc12b9-7e44-4ae5-9035-da04cd8b8e5d.png)

**Comment by LaserFrallan** [comment]
https://l.facebook.com/l.php?u=https%3A%2F%2Fshare.icloud.com%2Fphotos%2F05HfQqPLBDwoxwI_nXXhfLHtQ%3Ffbclid%3DIwAR24qFKLDz3YbRW7VxW-lqBgZ3fn8isuFWCs9Bo_SyNN3fVbqUoalWqWrqs&h=AT0P_1dm2xOTzxp0FaXzcoNHyFonjulhWOXq7lkLqy5zrq8yybcbLxT_lpdCQ8p5VUT5lsmmC_NyrkW8c4x-ZMWFe57VtXHOL8HPZwDylf04RKkXVKGE9AVaxmMU6t8-kTmVng

Do not know if this works but it is a short video showing the unit and the mother board. It lights up and it flashes accordingly and visible when getting commands.

**Reply by gmmanonymus111** [reply]
Side note about buttons: The first two buttons (`Reset` and `Unlock`) are clickable, always (when connected ofc.). Third (`Set zero`), is enabled if position is not the same as home. The next 3, (`Center`, `Corner` and `Frame`) needs an image opened to operate. The last two (`Focus`, `Blink`) is enabled when the machine is unlocked, and idle (not running a job). `Blink` needs to be pushed to light the laser, while focus is 'toggle-able'.

**Comment by LaserFrallan** [comment]
Here is a picture of my mother board.
I confirm that the unit is Twotrees TT-5.5 Laser Engraver 400x300 

![256812208_286648783383312_5329831034983601887_n](https://user-images.githubusercontent.com/94539219/142419715-28703de4-9229-4cc5-9c00-d76acc80eb99.jpg)

**Reply by gmmanonymus111** [reply]
Your laser seems to be plugged in to an incorrect slot. According to the image below, you plugged it into `probe`. It should go to `12V` out and `TTL out`
![image](https://user-images.githubusercontent.com/9658243/142424152-e2dbc259-acef-48c9-914a-389706e24da4.png)

**Reply by gmmanonymus111** [reply]
More precisely
![image](https://user-images.githubusercontent.com/9658243/142425430-7166e28a-1654-4d5b-bc7a-c9a50ccfc914.png)

**Comment by LaserFrallan** [comment]
Here is a screenshot of the button edit function.
I do not have same S figures but I did "play around" and tried to increase the values from 3% to 10% but no reaction from the laser at all.
![Uploading 11.png…]()

**Reply by gmmanonymus111** [reply]
Image upload failed

**Comment by LaserFrallan** [comment]
These?
![12](https://user-images.githubusercontent.com/94539219/142420673-73dfa78a-68d9-4d2f-8afa-926423d9f834.png)

**Comment by LaserFrallan** [comment]
![13](https://user-images.githubusercontent.com/94539219/142421174-c1bbbe60-1f6f-4994-b7bd-86a7d59b6b26.png)

This is what I get out. Cannot scroll upwards.

**Reply by gmmanonymus111** [reply]
`$120` and `$121` aceleration settings seem to be a little high. Try to lower it to around 600 at first (if you need more, you can increase it, then make a test run to see if arduino can handle it) To change, use either the GRBL config editor (`Grbl`->`Grbl Configuration`) or send `$120=600` and `$121=600` through comand input

**Comment by LaserFrallan** [comment]
Now I managed to scroll it to the top.
![14](https://user-images.githubusercontent.com/94539219/142421622-8b94e7b9-612e-4bd3-9eb0-a58e9eb238bb.png)

**Comment by LaserFrallan** [comment]
Confirmed!
Only Global button that does not light up.
Now, Blink and Focus still just not giving any action, even if buttons are lit up. Plus no errors.
Now if I click center, corner, frame, I do get the error "abort cycle"

I also tried to loosen the stepper cog wheels on all three motors, just to make sure they were not tight. No difference.

**Comment by LaserFrallan** [comment]
![15](https://user-images.githubusercontent.com/94539219/142422711-ac24f04c-2ec9-49a9-b7a5-38f46484a103.png)

**Comment by LaserFrallan** [comment]
That's IT!!!!
Took me a while before I figured out that this unit does not have another 12V cable more than the Power Supply.
I hade the unit powered up and the fan went on and off.
Plugged it in and out twice more and bang, it fired up.

THANKS!!!!

Afterthought: I should have reacted to the "probe" that was written on the circuit board. But the blue connector on both sides and being 3 pole, through me off. I am happy I did not damage anything.
